modules/openy_features/openy_media/modules/openy_media_image/openy_media_image.info.yml

```diff
@@ -20,3 +20,4 @@ dependencies:
   - entity_embed
   - focal_point
   - openy_media
+  - media_directories
```

Comment:
Why here? Why not for openy_media?
I bet this is not related to only images, but to all types